client/me/help/help-contact/index.jsx

```diff
@@ -190,12 +190,12 @@ class HelpContact extends React.Component {
 
 	submitKayakoTicket = ( contactForm ) => {
 		const { subject, message, howCanWeHelp, howYouFeel, site } = contactForm;
-		const { currentUserLocale, supportVariation, translate } = this.props;
+		const { currentUserLocale, supportVariation } = this.props;
 		let plan = 'N/A';
 		if ( site ) {
 			plan = `${ site.plan.product_name_short } (${ getPlanTermLabel(
 				site.plan.product_slug,
-				translate
+				( val ) => val // Passing an identity function instead of `translate` to always return the English string
 			) })`;
```

Comment:
Tested this LGTM.
As we discussed the fix may look a little bit hacky, but it is the simplest solution right now.

However a possible cleaner solution might be to break up the lib function into more meaningful sub-functions.
https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/plans/index.js#L508-L522

One which plainly sends a label. And another that depends on the earlier function but also localises the relevant string.
This will require some wider refactoring (of just the name) wherever `getPlanTermLabel` is used.

Reply:
That's a great suggestion! However, I feel that the refactoring would be outside the scope of this PR and can be done separately. I'll add this to my TODO list.